src/index.ts

```diff
@@ -116,16 +132,34 @@ export { useBreakpoint } from './hooks/useBreakpoint';
 export { useRequest } from './hooks/useRequest';
 export { createPoll, Poll } from './utils/poll';
 export { default as Theme } from './theme';
-export { Theme as ThemeType } from './common-types';
 
 export {
 	default as asRendition,
 	withStyledSystem,
 	withTooltip,
 } from './asRendition';
 
-import * as SchemaSieve from './components/Filters/SchemaSieve';
-export { SchemaSieve };
+export {
+	FULL_TEXT_SLUG,
+	Operator,
+	OperatorSlugs,
+	CreateFilter,
+	EditSchema,
+	FilterSignatureWithKey,
+	getSignatures,
+	createFilter,
+	createFullTextSearchFilter,
+	filter,
+	parseFilterDescription,
+	getOperators,
+	getModelFilter,
+	getSchemaFormat,
+	getPropertyScheme,
+	convertRefSchemeToSchemaPath,
+	getSchemaTitle,
+	generateSchemaFromRefScheme,
+	getSubSchemaFromRefScheme,
```

Comment:
I kind of preferred it as it was exposed under a special namespace, rather than having all those types & functions exported flat.